packages/core/contracts/financial-templates/expiring-multiparty/PricelessPositionManager.sol

```diff
@@ -604,6 +609,18 @@ contract PricelessPositionManager is FeePayer, AdministrateeInterface {
         return;
     }
 
+    function trimExcess(IERC20 token) external fees() nonReentrant() {
+        FixedPoint.Unsigned memory balance = FixedPoint.Unsigned(collateralCurrency.balanceOf(address(this)));
+        if (address(token) == address(collateralCurrency)) {
+            // If it is the collateral currency, send only the amount that the contract is not tracking.
+            // Note: this could be due to rounding error or balance-chainging tokens, like aTokens.
+            token.safeTransfer(excessTokenBeneficiary, balance.sub(pfc()).rawValue);
```

Comment:
So how does the balance of the contract diverge from `pfc()`? 
- user accidentally sends collateral to the contract
- interest denominated in collateral accrues to contract

anything else?

Reply:
I think those are the only two cases. But obviously, the viability of this change depends on that assumption. I'm going to take another pass through the contract to ensure there are no places where the balance of the contract is expected to deviate from pfc.